mlir/test/Conversion/ComplexToROCDL/complex-to-rocdl.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN: mlir-opt %s -convert-complex-to-rocdl -canonicalize | FileCheck %s
```

Comment:
Please make tests minimal by not running passes that are not being tested, such as the canonicalizer.